pkgs/npm/package.json

```diff
@@ -17,11 +17,10 @@
     "url": "https://github.com/DefangLabs/defang/issues"
   },
   "scripts": {
-    "build": "tsc",
-    "postinstall": "node ./bin/installer.js"
```

Comment:
Don't need a postinstall script as the cli.js will get the latest is not present.